viewer/src/public/types.ts

```diff
@@ -7,6 +7,12 @@ import { SectorGeometry } from '@/datamodels/cad/sector/types';
 import { SectorQuads } from '@/datamodels/cad/rendering/types';
 import { SectorCuller } from '@/internal';
 
+// we use these types in public API so they should be reexported here
+// to appear in the api reference docs
+export { CadRenderHints } from '../datamodels/cad/rendering/CadRenderHints';
```

Comment:
minor nitpick.. `@/datamodels`, but seeing as they are exported to experimentals it could be `@/experimental`. Anyways, that can be apart from this pr.

Reply:
Hm... Looks like typedoc can't resolve `@` alias. Will try to figure out on it a bit later then.